plugins/rtmp-services/data/services.json

```diff
@@ -6,12 +6,6 @@
             "name": "Twitch",
             "common": true,
             "stream_key_link": "https://dashboard.twitch.tv/settings/stream",
-            "supported video codecs": [
-                "h264"
-            ],
-            "supported audio codecs": [
-                "aac"
-            ],
```

Comment:
We just marked this field as required in our service submission guidelines, I’m pretty sure it’s intentionally explicit.

Reply:
This PR is meant to make them optional, because RTMP only use AAC. RTMP and FTL only use H264.

Those should maybe be required if the protocol is not RTMP or FTL, but RFC 45 is not reviewed so I'm stuck for PRing better.

And also I think everyone is not on the same wavelength about those that were added recently. Those are not required if we care to "add fallback"/"consider default" to H264 and AAC.

Reply:
I added the entries, it's not a big deal to make it optional imho. I'm fine with it for now. This is a relatively harmless file to change it later if we end up feeling different later. Plus I really don't want to block the beta over this.